views/rights.py

```diff
@@ -44,6 +48,7 @@ def get(self, request, *args, **kwargs):
                     ],
                 },
             }
+            logger.warning('Groups got an Attribute Error')
```

Comment:
Errors should go into `logger.error` in my opinion. It would be best if the team could decide framework-wide rules for severity of logs based on conditions.